source/YTEPs/YTEP-0037.rst

```diff
@@ -229,27 +234,24 @@ be done afterwards, and have been categorized below.
 
 * settle on a maximal line length and the status of ``unyt`` ("second" or third party)
```

Comment:
I think you should remove all mentions of unyt in this PR, since we agreed this ytep wasn't going to modify the way we handle unyt for now.

Reply:
This line is only about the isort configuration (where should unyt imports appear ? before or after numpy ?), it's not related to https://github.com/yt-project/yt/pull/2597 !

Reply:
Is it? This line talks about settling on whether unit is second or third party and it's in the roadmap section? It doesn't seem like this is isort specific?

Reply:
sorry I don't know how I got this wrong. Maybe I can blame GitHub on this.

Reply:
should be solved now

Reply:
Sorry to iterate on this, but can you explicitly add isort to your edit here so it is extremely clear we're talking about isort's import ordering? I'll make a suggestion.

Reply:
tbh I thought I did it already, thanks !